manifests/Quazar Conundrum.yaml

```
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/releases/tag/Major_ReleaseV015
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galxy spanning threat, changing hegemony and new factions rising to power.
```

```suggestion
description: The Quazar Conundrum is a massive century spanning story involving a galaxy-spanning threat, changing hegemony and new factions rising to power.
```
I would also recommend you note that it's still a work in progress, so maybe:
```suggestion
description: The WIP Quazar Conundrum will be a massive century spanning story involving a galaxy-spanning threat, changing hegemony and new factions rising to power.
```